pages/index.vue

```diff
@@ -25,7 +27,7 @@ const ratingValue = ref(5)
     <div class="flex w-4/6 max-w-lg flex-col">
       <SidebaseLogo class="w-72 fill-blue-200" />
       <h1 class="text-xl">
-        Welcome to <a href="https://github.com/sidestream-tech/sidebase" target="_blank">sidebase</a>!
+        Welcome to <a href="https://github.com/sidestream-tech/sidebase" target="_blank">sidebase {{ store.doubleCount }}</a>!
```

Comment:
Can we maybe show a "real-world" example for the store?
My suggestion would be to integrate the `a-rate` (where the user can set a star rating) with the store. 

Alternatively you can also add a button which increases the count (you can also display the count inside of the button) 😄